### DISCUSSION TITLE
Email & Mobile Icons

### DISCUSSION THREAD

**Discussion by ceyerkes** [twilio-labs/paste#2519] [Request: Icon] [answered]
Hello! I'm in the process of exploring how tables in SendGrid (specifically Marketing Campaigns) can be translated into Paste. With the addition of SMS in SG, we need a way to distinguish between email and SMS single sends in the table:

![Screen Shot 2022-07-08 at 10 59 36 AM](https://user-images.githubusercontent.com/87141551/178037437-fbbd88ae-8564-452e-9edc-9041710c5fba.png)

I'd like to request an email and mobile Paste icon please, thank you!

**Comment by nkrantz** [comment, marked as answer]
Hey @ceyerkes! The DSYS team has agreed that those icons make sense as a part of the design system. Please follow the [self-service instructions for adding the icons to Figma](https://paste.twilio.design/introduction/contributing/icons#setting-up-an-icon-in-figma), and let us know whether an engineer on your team will be able to add them to the repo following the instructions towards the bottom of that page. If not, we'll have to add it as a ticket to our backlog and might take a bit longer on our end to get it into the system.

**Reply by ceyerkes** [reply]
Will do, thanks Nora! Which icon style does Paste use from Streamline? This isn't super high priority at the moment, since SendGrid isn't in Paste yet, but I wanted the icons to be ready for when it is/while I get new Pastified designs ready.

**Reply by nkrantz** [reply]
Thanks for that additional context. For Streamline, you can start with [Core Line - Free](https://www.streamlinehq.com/icons/streamline-mini-line).

If you're not pressed for time, you can also wait until [this ticket](https://issues.corp.twilio.com/browse/DSYS-3330) is worked on by the DSYS team. The following icons will be added for Flex needs, so if they fit your needs too, feel free to plan to use them instead. 
<img width="485" alt="image" src="https://user-images.githubusercontent.com/75342690/178331696-2535ccfc-1bf6-4ae4-8794-a55c4c5ec2b1.png">

**Reply by ceyerkes** [reply]
Oooo, I think these would work perfectly for our needs on SG, I'm happy to wait - is there a rough timeline for when these might be added, just so I can make a note to check back/add them to my project? Thanks!

**Reply by nkrantz** [reply]
Ok, great! The icons are currently under the [September 6 release](https://paste.twilio.design/roadmap#2022-09-06-candidate). Feel free to also follow along on Jira.
